### DISCUSSION TITLE
--experimental vs --preview-feature

### DISCUSSION THREAD

**Discussion by Cretezy** [prisma/prisma#4104] [Q&A] [answered]
Why have 2 flags with the same purpose but with different names?

**Comment by thebiglabasky** [comment, marked as answer]
Hi,
We recently clarified terminology ([cf docs here for details](https://www.prisma.io/docs/more/releases)) around the different stages we may use when delivering products and features to ensure people know better what to expect. 
As a result there is a discrepancy between the current migrate still using experimental for backwards compatibility reasons and the newly introduced db push which is using the terminology we decided on going forward. 
Migrate is being significantly changed at the moment so the --experimental flag will go away as soon as we release the new version in preview which should happen in the coming weeks, so this in-between situation shouldn’t last for long.

**Comment by thebiglabasky** [comment]
If you want to try out the new migrate and see how it is going to change your workflow (provided you’re using it right now) feel free to chime in and take part in the [Early Access Program](https://prisma.slack.com/archives/C01739JGFCM/p1604078848016200)

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
